### PR TITLE
[Codelabs] Simplified version correction.

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

File: docs/codelabs/building-beautiful-sites/docsite/2-explore.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ While far from the end result we want, our skeleton site is looking pretty good
 
 ## Open index.html and app.css
 
-Copy and paste into Terminal `open -a Sublime app.css; open -a Sublime index.html`.
+In Terminal, press `command + T` to open another tab.
+
+Then copy and paste into the new tab `open -a Sublime app.css; open -a Sublime index.html`.
```
Comment (jonathanzong) [inline docs/codelabs/building-beautiful-sites/docsite/2-explore.md]: `open -a "Sublime Text" app.css index.html`

Comment (willlarche) [reply docs/codelabs/building-beautiful-sites/docsite/2-explore.md]: oh even better.

**Comment by jonathanzong** [comment]
Possible enhancement: add a link to the next step at the bottom of each page?

File: docs/codelabs/building-beautiful-sites/docsite/2-explore.md (hunk 1 above)
Comment (willlarche) [reply docs/codelabs/building-beautiful-sites/docsite/2-explore.md]: Done

**Comment by jonathanzong** [comment]
`[Next step: Set up your development environment](/web/codelab/1-setup/)`
`[Next step: Explore the starter code](/web/codelab/2-explore/)`
`[Next step: Explore the starter code](/web/codelab/2-explore/)`
`[Next step: Explore the starter code](/web/codelab/2-explore/)`
`[Next step: Explore the starter code](/web/codelab/2-explore/)`

**Comment by jonathanzong** [comment]
LGTM

**Pull request merged**